src/server/schemas/order.schema.ts

```diff
@@ -0,0 +1,41 @@
+import  mongoose  from "mongoose"
+import { Order } from "../../shared/models/order.model"
+const {Schema, model} = mongoose
+
+
+const orderSchema = new Schema<Order>({
+    orderItems: [{
+        type: mongoose.Schema.Types.ObjectId,
+        ref: 'OrderItem',
+        required: true
+      }],
+      shippingAddress1: {
```

Comment:
if you decide to move everything into a ShippingAddress interface, this page might get a bit tricky i would try `ShippingAddress: Schema.Types.Mixed `

Reply:
I'll take that into consideration thanks bryce